doc/install.rst

```diff
@@ -269,26 +269,34 @@ python library for Windows, Mac OSX and Linux.
 Anaconda offers scikit-learn as part of its free distribution.
 
 
-Intel conda channel
--------------------
+Intel Extension for Scikit-learn
+--------------------------------
 
-Intel maintains a dedicated conda channel that ships scikit-learn:
+Intel maintains an optimized x86_64 package, available in PyPI (via `pip`),
+and in the `main`, `conda-forge` and `intel` conda channels:
 
 .. prompt:: bash $
 
-  conda install -c intel scikit-learn
+  conda install scikit-learn-intelex
 
-This version of scikit-learn comes with alternative solvers for some common
-estimators. Those solvers come from the DAAL C++ library and are optimized for
-multi-core Intel CPUs.
+This package can accelerate an existing installation of scikit-learn and comes 
+with alternative solvers for some common estimators. Those solvers come 
```

Comment:
Well, it's ether not an alternative version of scikit as it can't be used as standalone package - scikit-learn package is required to work. 
Previous version was talking about daal4py package which does provide some estimators with scikit-learn like API. But with scikit-learn-intelex it's technically doing what we are saying here - based on existing scikit-learn instance replacing some estimator calls. 

Also from versioning and API changes perspective - user code should adhere to base scikit-learn version to to scikit-learn-intelex versioning.

Reply:
How about this then:

> This package has an Intel optimized version of many estimators. Whenever an
> alternative implementation doesn't exist, scikit-learn implementation is used as
> a fallback. The optimized solvers come from ....

Reply:
yes, let's use this version